webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -306,6 +306,15 @@
 		<servlet-name>jsp</servlet-name>
 		<url-pattern>*.jsp</url-pattern>
 	</servlet-mapping>
+
+	<servlet>
+		<servlet-name>default</servlet-name>
```

Comment:
See https://tomcat.apache.org/tomcat-7.0-doc/default-servlet.html.

This is why this PR has such a big impact. Any route not covered by any of the other servlets will end up hitting this route.